test/getStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, applyMiddleware, compose } from 'redux'
+import thunk from 'redux-thunk'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'redux-thunk'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->